foodkept/Startup.cs

```diff
@@ -63,6 +79,11 @@ public void ConfigureServices(IServiceCollection services)
 
         }
 
+        public void ConfigureContainer(ContainerBuilder builder)
+        {
+            builder.Register(x => Log.Logger).SingleInstance();
```

Comment:
What does this log?

Reply:
registers serilog as main logger, because default logger is some kind of built in asp.net which in not that good as serilog